minifi_main/CMakeLists.txt

```diff
@@ -74,12 +74,10 @@ if (WIN32)
     target_compile_definitions(minifiexe PUBLIC SERVICE_NAME="Apache NiFi MINiFi")
 endif()
 
-if (NOT WIN32)
-add_custom_command(TARGET minifiexe POST_BUILD
-    COMMAND cat ${CMAKE_BINARY_DIR}/all.log)
+if (WIN32)
+    add_custom_command(TARGET minifiexe POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:minifiexe> ${CMAKE_BINARY_DIR}/minifi_main/)
 else()
-#add_custom_command(TARGET minifiexe POST_BUILD
-#           COMMAND type ${CMAKE_BINARY_DIR}/all.log)
+    add_custom_command(TARGET minifiexe POST_BUILD COMMAND cat ${CMAKE_BINARY_DIR}/all.log)
```

Comment:
I think this `cat` can go, the contents of that log file is bogus anyway.

Reply:
Ah OK, I didn't realize this is the part where we print that standard processors have been disabled.  Removed in 98a74b7c1fbc044fcc899d63034cbe4354ab6265.